ansible/roles/master/defaults/main.yml

```diff
@@ -1,7 +1,13 @@
-kube_master_insecure_port: 8080
+kube_api_insecure_port: 8080
+
+kube_master_backend_api_port: 4443
```

Comment:
When using keepalived, we should be able to preserve default ports. For example, the kube-api front-end and backend can listen 443. This is because the kube-api front-end will listen on a different IP (the keepalived vip) than the backend, which listens to 443 of the actual ip.

In this scenario, we can simply use the existing port vars:

```ansible//roles/kubernetes/defaults/main.yml:kube_master_api_port: 443
ansible//roles/master/defaults/main.yml:kube_master_insecure_port: 8080``